GDJS/Runtime/pixi-renderers/pixi-image-manager.ts

```diff
@@ -463,6 +463,54 @@ namespace gdjs {
       }
       return particleTexture;
     }
+
+    dispose(): void {
+      this._loadedTextures.clear();
+
+      const threeTextures: THREE.Texture[] = [];
+      this._loadedThreeTextures.values(threeTextures);
+      this._loadedThreeTextures.clear();
+      for (const threeTexture of threeTextures) {
+        threeTexture.dispose();
+      }
+
+      const threeMaterials: THREE.Material[] = [];
+      this._loadedThreeMaterials.values(threeMaterials);
+      this._loadedThreeMaterials.clear();
+      for (const threeMaterial of threeMaterials) {
+        threeMaterial.dispose();
+      }
+
+      const diskPixiTextures = [...this._diskTextures.values()];
+      this._diskTextures.clear();
+      for (const pixiTexture of diskPixiTextures) {
+        if (pixiTexture.destroyed) {
+          continue;
+        }
+
+        pixiTexture.destroy();
+      }
```

Comment:
Will this work?
```
      for (const pixiTexture of this._diskTextures.values()) {
        if (pixiTexture.destroyed) {
          continue;
        }
        pixiTexture.destroy();
      }
      this._diskTextures.clear();
```
Minus new array creation.

Reply:
done